src/geo/crs/CRS.js

```diff
@@ -73,6 +73,9 @@ L.CRS = {
 		return L.bounds(min, max);
 	},
 
+	// @method distance(latlng1: LatLng, latlng1: LatLng): Number
+	// Returns the distance between two geographical coordinates.

```

Comment:
distance is not necessarily in meters, only for L.CRS.Earth and it's inheritants. CRS.Simple is just euclidean distance.